jobbergateapi2/pagination.py

```diff
@@ -8,13 +8,12 @@ class Pagination:
     Basic pagination class
     """
 
-    def __init__(self, q: str = None, limit: int = 10, offset: int = 0):
+    def __init__(self, limit: int = 10, skip: int = 0):
         """
-        Pagination constructor using default limit and offset
+        Pagination constructor using default limit and skip offset.
```

Comment:
When we talk about limit and offset, inside the context of pagination, their meaning must be already clear, so I don't think this add much, but instead it could create some short of confusion to see an unexpected comment for simple things, when nowhere else has them